ci-operator/step-registry/openshift/microshift/e2e/metal-tests/openshift-microshift-e2e-metal-tests-commands.sh

```diff
@@ -3,6 +3,59 @@ set -xeuo pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
+finalize() {
+  scp -r "${INSTANCE_PREFIX}:/home/${HOST_USER}/microshift/_output/test-images/scenario-info" "${ARTIFACT_DIR}"
+
+  STEP_NAME="${HOSTNAME##${JOB_NAME_SAFE}-}"
+  REPORT="${ARTIFACT_DIR}/custom-link-tools.html"
+  cat >>${REPORT} <<EOF
+<html>
+<head>
+  <title>Test logs</title>
+  <meta name="description" content="Links to relevant logs">
+  <link rel="stylesheet" type="text/css" href="/static/style.css">
+  <link rel="stylesheet" type="text/css" href="/static/extensions/style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,700" rel="stylesheet">
+  <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.indigo-pink.min.css">
+  <link rel="stylesheet" type="text/css" href="/static/spyglass/spyglass.css">
+  <style>
+    a {
+        display: inline;
+        color: #FFFFFF;
+    }
+    a:hover {
+      text-decoration: underline;
+    }
+    p {
+      display: inline;
+      color: #FFFFFF;
```

Comment:
Is it possible to add a background color that matches the grey background color in the prow preview? When you view just the html alone in the artifacts you can't see the links since the background is also white

Reply:
Or maybe, just make the background white so the prow preview renders that?

Reply:
Changed the background color so it shows dark too outside of prow.